tooling/nargo_fmt/tests/expected/benchmarking_contract.nr

```diff
@@ -0,0 +1,78 @@
+// A contract used for running benchmarks.
+// We should try to change this contract as little as possible, since any modification
+// would alter the metrics we're capturing in the benchmarks, and we want to keep the
+// subject being tested as unmodified as possible so we can detect metric changes that 
+// arise from code changes.
```

Comment:
Can you change this comment? It looks like it was copy pasted from an aztec contract but it reads as if we shouldn't change this fmt test because it'll be used as benchmarking somehow. Let's rename the name of the file while we're at it - that doesn't need to mention benchmarking either.